src/sentry/tasks/commit_context.py

```diff
@@ -143,10 +140,10 @@ def process_commit_context(
             except ApiError:
                 logger.info(
                     "process_commit_context_all_frames.retry",
-                    extra={**basic_logging_details, "retry_count": self.request.retries},
```

Comment:
Taskworkers don't expose a thread local/proxy for getting the current task data.  This is the only place we use this in sentry + getsentry, so I don't see us implementing it in the near term.